game-core/src/main/java/games/strategy/triplea/delegate/AbstractBattle.java

```diff
@@ -60,6 +63,10 @@
 
   protected final GameData m_data;
 
+
+  @Getter(AccessLevel.PROTECTED)
+  private final BattleRules battleRules;
```

Comment:
Did you mean to mark this field `transient`?  `BattleRules` is not serializable.  (A `NotSerializableException` in this scenario seems to be logged quietly instead of triggering the error dialog.  Not sure why.)

Also note that you'll have to deal with a `null` field value here when loading an old save game.  Might be time for that `readObject()` workaround we talked about in another thread.

Reply:
I lost context on `BattleRules`, seems like some other usages of it were wiped out. I removed it:  3690171